test/unit/puppetlabs/services/jruby_pool_manager/jruby_pool_test.clj

```diff
@@ -219,6 +220,54 @@
    (jruby-testutils/jruby-config {:max-active-instances max-instances
                                   :max-borrows-per-instance max-borrows})))
 
+(defn drain-and-refill
+  [pool-context
+   instance-count]
+  (let [instances (jruby-testutils/drain-pool pool-context 2)
```

Comment:
2 being hardcoded seems surprising. I'm also not sure what it is for, even having read the `drain-pool` function.